docs/quickstart.md

```diff
@@ -0,0 +1,760 @@
+    logger.setLevel(os.getenv("LOG_LEVEL", "INFO"))
+
+    app = ApiGatewayResolver()
+
+
+    @app.get("/hello/<name>")
+    def hello_name(name):
+        logger.info(f"Request from {name} received")
+        return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+    @app.get("/hello")
+    def hello():
+        logger.info(f"Request from unknown received")
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    def lambda_handler(event, context):
+        logger.debug(event)
+        return app.resolve(event, context)
+    ```
+=== "requirements.txt"
+
+    ```bash
+    aws-lambda-powertools
+    python-json-logger
+    ```
+So in our first attempt we create new APP Logger. We configure Log Handler, formatter and log level. Then we use this logger to log required information. We will see logs structured like that
+```json
+{"asctime": "2021-11-22 15:32:02,145", "levelname": "INFO", "name": "APP", "message": "Request from unknown received"}
+```
+instead of
+```json
+[INFO]  2021-11-22T15:32:02.145Z        ba3bea3d-fe3a-45db-a2ce-72e813d55b91    Request from unknown received
+```
+
+So far so good! You might also decide to add additional context into your logs.
+You can take it from lambda context or event. Downside of this approach is that we would add additional complexity to ensure that you always add those specific attributes in every place.
+Also, we added boilerplate code and even with additional third party library it is not straightforward. 
+Can we do better? Yes! We can use powertools logger capabilities as with next example.
+=== "app.py"
+
+    ```python hl_lines="1 6 13 19 23"
+    from aws_lambda_powertools import Logger
+    from aws_lambda_powertools.logging import correlation_paths
+
+    from aws_lambda_powertools.event_handler.api_gateway import ApiGatewayResolver
+
+    logger = Logger(service="APP")
+
+    app = ApiGatewayResolver()
+
+
+    @app.get("/hello/<name>")
+    def hello_name(name):
+        logger.info(f"Request from {name} received")
+        return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+    @app.get("/hello")
+    def hello():
+        logger.info(f"Request from unknown received")
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    @logger.inject_lambda_context(correlation_id_path=correlation_paths.API_GATEWAY_REST, log_event=True)
+    def lambda_handler(event, context):
+        return app.resolve(event, context)
+    ```
+
+We imported powertools Logger class which we created instance from in line 6. And this is basically it! By using powertools logger we have all required configuration done out of the box. 
+We used `logger.inject_lambda_context` decorator to inject lambda context into every log. We also instruct logger to log correlation id taken from API Gateway and to log coming event automatically. 
+Correlation id will help with tracking request between api gateway and lambda.
```

Comment:
How does it help?

Reply:
Rephrased to:
`Because powertools library adds a correlation identifier to each log, we can easily correlate all the logs generated for a specific request.`